messages/en.json

```
@@ -240,18 +279,24 @@
 	"importFromDelegator": "Import conference from Delegator",
+	"importWizardSubtitle": "Import conference",
 	"imprintAndPrivacy": "Imprint &amp; Privacy Policy",
 	"informalCaucus": "Informal meeting",
 	"insertAfterPresentation": "Insert after OP.{index}",
 	"insertAsFirstClause": "Insert as first clause",
 	"insertAtBeginning": "Insert at beginning",
 	"invalidShareCode": "Invalid share code",
 	"italic": "Italics",
+	"jumpToStep": "Jump",
 	"language": "Language",
 	"launcher": "Launcher",
```

Biome (2.4.14)

[error] The key launcher was already declared.

(lint/suspicious/noDuplicateObjectKeys)